bebop_driver/launch/bebop_node.launch

```diff
@@ -10,6 +10,6 @@
             <param name="bebop_ip" value="$(arg ip)" />
             <rosparam command="load" file="$(arg config_file)" />
         </node>
-        <include file="$(find bebop_description)/launch/description.launch" />
+        <!--include file="$(find bebop_description)/launch/description.launch" /-->
```

Comment:
Could you please revert back this line? I understand that Bebop's URDF does not match Disco's, but that should be fixed in a more generalized way.